app/controllers/api/v1/accounts/contacts/conversations_controller.rb

```diff
@@ -8,9 +8,7 @@ def index
   private
 
   def inbox_ids
-    if Current.user.administrator?
-      Current.account.inboxes.pluck(:id)
-    elsif Current.user.agent?
```

Comment:
If the user is an admin, I guess we could show every inbox available.

Reply:
`assigned_inboxes` handle this condition. it returns all inboxes for admin users and only the specific inboxes for agents